bluebottle/analytics/management/commands/export_analytics_data.py

```diff
@@ -62,17 +62,23 @@ def handle(self, **options):
     @override_settings(ANALYTICS_ENABLED=True)
     def _process(self, cls, start_date, end_date):
         cls_name = cls.__name__
+        # TODO: Come up with a better way of generically getting the timestamp for influxdb log entry
```

Comment:
We might event want to change the way we set timestamps on different models. Having consistent naming and logic on when, what timestamp is set. For now this will do.